include/uv-darwin.h

```diff
@@ -52,6 +57,7 @@
   void* cf_member[2];                                                         \
   int cf_error;                                                               \
   uv_mutex_t cf_mutex;                                                        \
+  enum uv_fs_event_type_flag type_flag;                                       \
```

Comment:
Well, that's not it. https://github.com/libuv/libuv/pull/641/files#diff-73c6612c3129d1050a5989300eb7c4adR260 is required as well to maintain compatibility. And we must only do that if what we are watching is a file